add modally present and test

sergiusKurganov opened the pull request:
**WHAT WAS ADDED ?**
A **modal presentation style** has been added to show the view controller. **Unit test** was also added. 
**FOR WHAT WAS ADDED ?**
This feature was necessary for the **modal presentation** display of controllers on the guidelines of Apple. Without this, you need to do the **Router extension**.

sergiusKurganov committed:
add modally present and test

ferranabello commented:
Hi @sergiusKurganov,
thanks for the PR!

I created another PR #90 adding more functionality to meet what we have for all the core router functions at the moment. I also renamed the new function to **present**. I was about to ask you to do too many changes. It's faster this way 👍 

I also added more deep testing for this new functionality and updated the example using the new router function.

If you like my solution, then we'll merge #90 and close this PR.

Cheers!

sergiusKurganov commented:
> Hi @sergiusKurganov,
> thanks for the PR!
> 
> I created another PR #90 adding more functionality to meet what we have for all the core router functions at the moment. I also renamed the new function to **present**. I was about to ask you to do too many changes. It's faster this way 👍
> 
> I also added more deep testing for this new functionality and updated the example using the new router function.
> 
> If you like my solution, then we'll merge #90 and close this PR.
> 
> Cheers!

Ok! I'll do a git pull and join to you #90 PR ;)
Thank you)

ferranabello commented:
This will be implemented in #90
